<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerMainWindow</class>
 <widget class="QMainWindow" name="ManagerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ReturnBook">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>还书</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SingleInput">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>单册入库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MultiInput">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>多册入库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UserManage">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>用户管理</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>用户借书记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>651</width>
      <height>541</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;fd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>借书记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>根据学生查询</string>
     </attribute>
     <widget class="QPushButton" name="StuQueryButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stu_name">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="stu_sno">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="stu_tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>661</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>根据书查询</string>
     </attribute>
     <widget class="QLineEdit" name="bookname">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="bookbno">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="book_tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>651</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BookQueryButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
